pkgs/servers/alps/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromSourcehut }:
+
+buildGoModule rec {
+  pname = "alps";
+  version = "d4c35f3c3157bece8e50fd95f2ee1081be30d7ae";
```

Comment:
Version should be 2021-09-29, and the revision moved to the fetcher. Same with the PR title and commit please

Reply:
Thank you! I've pushed the changes